encrypted-media/scripts/setmediakeys-multiple-times-with-different-mediakeys.js

```diff
@@ -0,0 +1,89 @@
+function runTest(config, qualifier) {
+    var testname = testnamePrefix( qualifier, config.keysystem )
+                         + ', setmediakeys multiple times with different mediakeys';
+
+    var configuration = getSimpleConfigurationForContent( config.content );
+
+    if ( config.initDataType && config.initData ) {
+        configuration.initDataTypes = [ config.initDataType ];
+    }
+
+    async_test (function (test) {
+        var _video = config.video,
+            _access,
+            _mediaKeys1,
+            _mediaKeys2,
+            fail;
```

Comment:
It is set to true or false based on whether one of the promises is resolved or rejected and then only tested in a downstream then function. So, I don't think it is necessary to initialize it.